### PR TITLE
Add swap bait with lure menu entry swap config for fishing spot

### PR DESCRIPTION
**Pull request by pwatts6060**

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java [hunk 1]
```diff
@@ -276,6 +276,17 @@ default boolean swapHarpoon()
 		return false;
 	}
 
+	@ConfigItem(
+			keyName = "swapBait",
+			name = "Bait",
+			description = "Swap Lure with with Bait on Fishing spot",
+			section = objectSection
```
Comment (dekvall) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java:283]: Dedent these lines.

**Review by dekvall — CHANGES_REQUESTED** [review]

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java [hunk 2]
```diff
@@ -238,6 +238,8 @@ void setupSwaps()
 		swap("big net", "harpoon", config::swapHarpoon);
 		swap("net", "harpoon", config::swapHarpoon);
 
+		swap("lure", "bait", config::swapBait);
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java:241]: Cave eels also have "Bait" options, as do some mystery Net / Bait spots. They should be covered the same way as the harpoon spots.

**Review by Nightfirecat — CHANGES_REQUESTED** [review]
This also needs to be rebased on master.

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java [hunk 3]
```diff
@@ -287,6 +287,17 @@ default boolean swapHarpoon()
 		return false;
 	}
 
+	@ConfigItem(
+		keyName = "swapBait",
+		name = "Bait",
+		description = "Swap Small Net, Net, Lure with with Bait on Fishing spot",
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java:293]: I'll commot this with GitHub's UI
```suggestion
		description = "Swap Small Net, Net, Lure with Bait on Fishing spot",
```

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java [hunk 4]
```diff
@@ -287,6 +287,17 @@ default boolean swapHarpoon()
 		return false;
 	}
 
+	@ConfigItem(
+		keyName = "swapBait",
+		name = "Bait",
+		description = "Swap Small Net, Net, Lure with Bait on Fishing spot",
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java:293]:
```suggestion
		description = "Swap Lure, Small Net with Bait on Fishing spot",
```

**Pull request merged**